libs/langgraph/langgraph/prebuilt/tool_node.py

```diff
@@ -34,6 +34,7 @@
 from langchain_core.runnables.utils import Input
 from langchain_core.tools import BaseTool, InjectedToolArg
 from langchain_core.tools import tool as create_tool
+from langchain_core.tools.base import _get_all_basemodel_annotations
```

Comment:
would be nice to make this a public API

Reply:
might need to just copy it over if we don't want to bump core dep

Reply:
i think it's fine to bump core dep